tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile

```diff
@@ -23,9 +23,12 @@ WORKDIR /openedx/ecommerce
 # nodejs requirements
 RUN npm install
 RUN ./node_modules/.bin/bower install --allow-root
-
+# Setup minimal yml config file, which is required by production settings
+RUN echo "{}" > /openedx/config.yml
+ENV ECOMMERCE_CFG /openedx/config.yml
```

Comment:
The `config.yml` file should not be required by the assets settings, as discussed [here](https://github.com/overhangio/tutor-ecommerce/issues/2#issuecomment-528224593).